Makefile.am

```diff
@@ -225,6 +225,40 @@ EXTRA_DIST += \
   javascript/ola/port_table.js \
   javascript/ola/universe_control.js \
   javascript/ola/universe_item.js \
+  javascript/new-src/README.md \
+  javascript/new-src/bower.json \
+  javascript/new-src/src/js/app.js \
```

Comment:
EXTRA_DIST is only if you want them included into the final distribution tarball. I can see arguments both ways so I'll leave it to Dave to decide.